ui/scss/component/_claim-list.scss

```diff
@@ -970,24 +970,26 @@
     }
   }
 
-  &:hover {
-    cursor: pointer;
+  @media (hover) {
```

Comment:
This breaks s-pen hover :(
Will most likely break vnc hover as well (usually parity, but not always)

Can confine the change to Safari through a classname or something?